package.json

```diff
@@ -44,7 +44,7 @@
     "prettier": "^2.1.1",
     "serverless": "1.x",
     "serverless-dotenv-plugin": "^3.0.0",
-    "serverless-offline": "^6.8.0",
+    "serverless-offline": "^6.9.0",
```

Comment:
Why was it necessary to update `serverless-offline`? This is something that would make sense to me in #58 instead.

Reply:
Is that the exact error message? The provider would have needed to be `nodejs14.x` (with the "js"). All looks fine in your PR though so I'm happy for you to merge 🙂